### PR TITLE
REF clean up the masked fraction cuts on SE images

### PR DESCRIPTION
**Pull request by beckermr**
This PR cleans up the masked fraction cuts.

### CODE REVIEW

File: config_files/des-pizza-slices-y6-v5.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  # if using piff in DES and a release earlier than Y6,
+  # you need to set the piff_run above too
+  psf_type: piff
+
+  # which SE WCS to use - one of piff, pixmappy or image
+  wcs_type: pixmappy
+
+  reject_outliers: False
+  symmetrize_masking: True
+  copy_masked_edges: False
```
Comment (beckermr) [inline config_files/des-pizza-slices-y6-v5.yaml:45]:
```suggestion
  copy_masked_edges: True
```

Comment (beckermr) [reply config_files/des-pizza-slices-y6-v5.yaml:45]: @esheldon did you turn this off?

Comment (beckermr) [reply config_files/des-pizza-slices-y6-v5.yaml:45]: ack nvm. I never turned it on!

**Comment by beckermr** [comment]
@esheldon This one is ready for review.

File: pizza_cutter/des_pizza_cutter/_coadd_slices.py [hunk 2]
```diff
@@ -138,12 +135,6 @@ def _build_slice_inputs(
         any pixels with any of the noise_interp_flags. It is the fraction of
         the subset of the SE image that approximatly overlaps the final coadded
         region.
-    max_unmasked_trail_fraction : float
```
Comment (esheldon) [inline pizza_cutter/des_pizza_cutter/_coadd_slices.py:141]: how do you know the trail exists but was unmasked?

**Review by esheldon — CHANGES_REQUESTED** [review]

File: pizza_cutter/des_pizza_cutter/_coadd_slices.py (hunk 2 above)
Comment (beckermr) [reply pizza_cutter/des_pizza_cutter/_coadd_slices.py:141]: This cut never really worked which is why it is being removed.

File: config_files/des-pizza-slices-y6-v5.yaml [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  #  - 16384 # TAPEBUMP
+
+  bad_image_flags:
+    - 0
+
+gaia_star_masks:
+  # multiply the radii by this factor
+  radius_factor: 1.75
+
+  # don't mask stars with gaia g mag less than this
```
Comment (beckermr) [inline config_files/des-pizza-slices-y6-v5.yaml:96]:
```suggestion
  # don't mask stars with gaia g fainter less than this
```

**Comment by beckermr** [comment]
@esheldon ready for another look!

File: config_files/des-pizza-slices-y6-v5.yaml [hunk 4]
```diff
@@ -93,7 +93,7 @@ gaia_star_masks:
   # multiply the radii by this factor
   radius_factor: 1.75
 
-  # don't mask stars with gaia g mag less than this
+  # don't mask stars with gaia g fainter less than this
```
Comment (esheldon) [inline config_files/des-pizza-slices-y6-v5.yaml:96]: remove "less"

**Review by esheldon — CHANGES_REQUESTED** [review]

File: config_files/des-pizza-slices-y6-v5.yaml [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+  #  - 16384 # TAPEBUMP
+
+  bad_image_flags:
+    - 0
+
+gaia_star_masks:
+  # multiply the radii by this factor
+  radius_factor: 1.75
+
+  # don't mask stars with gaia g fainter less than this
```
Comment (beckermr) [inline config_files/des-pizza-slices-y6-v5.yaml:96]:
```suggestion
  # don't mask stars with gaia g fainter than this
```

**Comment by beckermr** [comment]
OK. Ready for another look!

**Review by esheldon — APPROVED** [review]

**Pull request merged**